app/src/androidTest/java/net/skyscanner/backpack/chip/BpkChipTest.kt

```diff
@@ -37,31 +37,31 @@ class BpkChipTest : BpkSnapshotTest() {
   fun screenshotTestDefault() {
     val view = BpkChip(testContext)
     view.text = "tag"
-    snap(view)
+    snap(view.wrapInParent())
```

Comment:
I decided to wrap these in a parent as otherwise it would increase the size to the parent and wouldn't be an accurate representation of the chip - this makes it easier to verify the screenshots are correct in the future